doc/markus-contributors.txt

```diff
@@ -26,6 +26,7 @@ Anthony Labaere
 Anthony Le Jallé
 Anton Braverman
 Anushiya Antony
+Anubha Joshi
```

Comment:
This should appear before the preceding entry (for Anushiya)

Reply:
fixed!